lighthouse-core/report/styles/report.css

```diff
@@ -50,6 +50,8 @@ span, div, p, section, header, h1, h2, li, ul {
   --report-width: 1280px;
   --report-menu-width: 280px;
   --report-header-height: 58px;
+
+  --config-panel-header-height: 50px;
```

Comment:
I moved the config panel HTML and related CSS into a partial. Some position related CSS code is still in the main CSS file (now in `.report-body__fixed-body-container {...}`) so the CSS in the partial does not need to use the variables declared in the main CSS file.
Thx for your advice.